hackathon_site/dashboard/frontend/src/components/dashboard/TeamCardAdmin/TeamCardAdmin.module.scss

```diff
@@ -0,0 +1,23 @@
+@import "./../../../assets/abstracts/mixins";
+@import "./../../../assets/abstracts/variables";
+
+.name {
+    padding: 6px 0px;
+}
+
+.title {
+    padding: 0px 0px;
+}
+
+.content_container {
+    padding: 15px 25px;
+}
+
+.border {
```

Comment:
remove .border and .lastRow classes because they aren't being used

Reply:
removed!